app/components/primer/underline_nav_component.rb

```diff
@@ -5,27 +5,45 @@ module Primer
   # underlined selected state, typically used for navigation placed at the top
   # of the page.
   class UnderlineNavComponent < Primer::Component
+    include ViewComponent::SlotableV2
+
     ALIGN_DEFAULT = :left
     ALIGN_OPTIONS = [ALIGN_DEFAULT, :right].freeze
 
-    with_content_areas :body, :actions
+    #  Use the body for the navigation items
+    #
+    # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+    renders_one :body, lambda { |**system_arguments|
+      system_arguments[:classes] = class_names("UnderlineNav-body", "list-style-none", system_arguments[:classes])
+
+      Primer::BaseComponent.new(tag: :ul, **system_arguments) { content }
```

Comment:
hmm, what happens if someone passes `tag` through  `system_arguments`?
Maybe this is safer:

```suggestion
      system_arguments[:classes] = class_names("UnderlineNav-body", "list-style-none", system_arguments[:classes])
      system_arguments[:tag] = :ul

      Primer::BaseComponent.new(**system_arguments) { content }
```

Reply:
This should be `system_arguments[:tag] ||= :ul`, no?

Reply:
hmm yeah you're right